apps/frontend/src/modules/auth/auth-api.ts

```diff
@@ -36,6 +38,28 @@ class AuthApi extends BaseHTTPApi {
 		return await response.json<UserDto>();
 	}
 
+	public async requestResetPassword(payload: EmailDto): Promise<null> {
+		await this.load(this.getFullEndpoint(AuthApiPath.FORGOT_PASSWORD, {}), {
+			contentType: ContentType.JSON,
+			hasAuth: false,
+			method: "POST",
+			payload: JSON.stringify(payload),
+		});
+
+		return null;
```

Comment:
Let's return boolean from Backend (success = true, failure = false)

Reply:
Why do we need to send a boolean here?